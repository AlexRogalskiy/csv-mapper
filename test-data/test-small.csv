id,first_name,last_name,email,country,ip_address,Some Date,isbn
1,Thomas,Hart,thart@mita.biz,Iraq,143.117.60.33,2013-10-28,606558580-7
2,Carol,Burton,cburton@quatz.net,Indonesia,58.58.223.16,2013-11-16,305315666-0
3,Jeremy,Bowman,jbowman@voolia.org,Antarctica,152.43.12.144,2013-11-27,562485206-1
4,Helen,Mcdonald,hmcdonald@tazz.gov,Nicaragua,215.174.1.136,2013-04-07,643800006-X
5,Jimmy,Butler,jbutler@blogpad.info,South Africa,75.228.179.212,2013-03-10,266276572-0
6,Patricia,Gilbert,pgilbert@feedfish.name,Vanuatu,237.250.203.2,2013-11-03,631661305-9
7,Jacqueline,Gordon,jgordon@shufflebeat.gov,Peru,101.104.49.207,2013-05-24,673152442-5
8,Evelyn,Gutierrez,egutierrez@cogibox.org,Peru,162.102.77.69,2013-06-29,286409900-4
9,Sarah,Fuller,sfuller@buzzster.name,Ireland,200.27.43.210,2013-10-23,894654430-9
10,Bobby,Barnes,bbarnes@feedfish.org,Rwanda,141.157.154.54,2013-09-22,863255478-6
11,Russell,Fowler,rfowler@camido.biz,Uzbekistan,215.131.105.87,2013-10-11,264038858-4
12,Nicole,Jackson,njackson@quaxo.name,Nepal,101.40.167.134,2013-10-15,766501958-7
13,Shawn,Stewart,sstewart@agimba.edu,Zambia,135.241.11.222,2013-10-20,665204681-4
14,Janice,Shaw,jshaw@twinder.com,Isle of Man,147.80.145.12,2013-02-18,933839145-0
15,Amy,Wagner,awagner@devpulse.mil,Croatia,32.138.195.183,2013-10-03,452697010-7
16,Victor,Morales,vmorales@brainsphere.org,Christmas Island,133.98.105.129,2013-05-02,835814165-3
17,Gary,Cox,gcox@aimbu.net,Vanuatu,56.147.204.73,2013-11-02,101622872-4
18,Jessica,Harris,jharris@oloo.org,Greenland,159.241.8.102,2013-11-29,508512120-1
19,Steve,Rodriguez,srodriguez@gigazoom.name,Cameroon,161.56.190.44,2013-05-30,909830782-5
20,Lois,Weaver,lweaver@rhynyx.mil,Zambia,167.215.28.125,2013-09-23,869546377-X
21,Jeffrey,Coleman,jcoleman@zoovu.net,"Palestinian Territory, Occupied",215.77.156.195,2013-04-20,942812187-8
22,Timothy,Green,tgreen@snaptags.name,Marshall Islands,171.157.139.120,2013-05-26,299576929-1
23,Ruth,Lynch,rlynch@skimia.biz,Cook Islands,181.160.45.59,2013-10-02,566885935-5
24,Joyce,Larson,jlarson@voomm.name,Barbados,125.148.182.93,2013-10-20,731957492-6
25,Jeremy,Crawford,jcrawford@oozz.biz,Somalia,58.253.246.51,2013-08-07,394783299-0
26,Jimmy,Hansen,jhansen@quatz.org,Nigeria,128.195.214.18,2014-01-05,194156674-X
27,Lillian,Wagner,lwagner@browsebug.org,Slovenia,168.142.6.99,2014-01-12,989518952-4
28,Heather,Anderson,handerson@katz.mil,Niger,1.203.60.241,2013-02-17,440999130-2
29,Jimmy,Gordon,jgordon@brightbean.net,"Korea, South",171.21.150.18,2014-01-23,689818246-3
30,Paula,Larson,plarson@kayveo.info,Sweden,195.164.190.130,2014-01-29,892385973-7
31,Michelle,Parker,mparker@yamia.gov,Croatia,225.250.0.146,2013-09-12,979503850-8
32,Daniel,Fisher,dfisher@eidel.info,Antarctica,98.239.51.81,2013-05-08,442305590-2
33,Samuel,Lane,slane@kazio.org,San Marino,92.44.84.72,2013-09-12,400346331-5
34,Laura,Harper,lharper@miboo.mil,USSR,19.154.176.68,2013-07-12,006236708-0
35,Virginia,Smith,vsmith@kwimbee.mil,Gambia,221.219.72.89,2013-03-10,543626250-3
36,Louis,Martinez,lmartinez@bubbletube.mil,Solomon Islands,39.160.216.186,2013-11-22,452604201-3
37,Dorothy,Hernandez,dhernandez@innoz.edu,Uzbekistan,196.3.33.124,2014-02-03,408161537-3
38,Frances,Marshall,fmarshall@oozz.gov,Angola,80.108.134.161,2013-10-15,879622742-7
39,Andrew,Bowman,abowman@agivu.mil,Niger,58.149.0.198,2013-02-20,883497756-4
40,Richard,Mcdonald,rmcdonald@eimbee.com,Cape Verde,186.40.175.50,2014-01-25,641777485-6
41,Judy,Peterson,jpeterson@livepath.org,Jamaica,54.49.136.154,2013-09-20,224445443-6
42,Kimberly,Edwards,kedwards@talane.gov,Turks and Caicos Islands,40.109.195.126,2013-05-13,797243890-0
43,Daniel,Peterson,dpeterson@linkbridge.name,Mauritania,60.106.241.220,2014-01-11,723491135-9
44,Walter,Williams,wwilliams@meembee.com,Yugoslavia,112.152.8.1,2013-09-19,606158123-8
45,Margaret,Scott,mscott@zazio.net,Kiribati,152.134.226.83,2013-07-22,126644029-1
46,Kathleen,Jenkins,kjenkins@meezzy.net,Barbados,246.36.205.195,2013-07-29,863754134-8
47,Alice,Sullivan,asullivan@meezzy.org,Brunei Darussalam,101.249.196.118,2013-04-22,356075037-7
48,Brandon,Ramirez,bramirez@blogpad.gov,Fiji,65.16.99.230,2013-05-05,483004106-4
49,Melissa,Price,mprice@youtags.com,Spain,207.240.15.14,2013-03-29,353897312-1
50,Pamela,Olson,polson@leexo.com,Aruba,97.71.127.233,2014-01-04,648204364-X
51,Jesse,Patterson,jpatterson@topicware.edu,San Marino,117.57.51.98,2013-11-24,679296503-0
52,Paul,Jacobs,pjacobs@yakidoo.biz,Slovakia,198.181.174.214,2013-03-03,076244837-7
53,Jeremy,Matthews,jmatthews@abata.biz,Norfolk Island,147.97.129.199,2013-02-18,890081868-6
54,Ruth,Campbell,rcampbell@browsedrive.edu,Lesotho,218.157.177.32,2013-07-06,148019645-2
55,Ryan,Cunningham,rcunningham@plambee.edu,Benin,202.51.54.192,2013-03-28,391670972-0
56,Susan,Graham,sgraham@mynte.mil,Guernsey,91.56.95.177,2014-02-09,948180579-4
57,Peter,Banks,pbanks@shuffletag.name,Croatia,1.197.238.225,2013-03-19,798686576-8
58,Karen,Hanson,khanson@camido.name,Tonga,126.241.60.171,2013-07-28,435829572-X
59,Billy,Hunt,bhunt@rhyloo.mil,Greece,89.23.100.90,2013-08-11,194549716-5
60,Tammy,Bradley,tbradley@quinu.edu,Egypt,24.150.118.160,2014-01-31,119738861-3
61,Antonio,Diaz,adiaz@fatz.gov,Trinidad and Tobago,37.170.44.231,2013-11-20,693386375-1
62,Shirley,Jones,sjones@flashdog.gov,Palau,224.124.196.36,2014-02-14,753365783-7
63,Deborah,Ryan,dryan@voonix.info,Equatorial Guinea,247.23.164.36,2013-12-25,030270463-9
64,William,Berry,wberry@chatterbridge.net,Qatar,29.73.85.127,2013-08-15,592041918-0
65,Ruth,Bishop,rbishop@tagtune.net,Sierra Leone,79.192.244.45,2013-06-03,134566421-4
66,Virginia,Howell,vhowell@babbleset.gov,Zambia,241.65.131.207,2014-01-15,269845674-4
67,Kenneth,Rivera,krivera@kwimbee.info,Oman,190.230.228.143,2013-11-17,882603825-2
68,Amanda,Wells,awells@wikibox.info,Austria,241.193.93.250,2013-04-01,102983738-4
69,Ralph,Shaw,rshaw@trudeo.biz,Puerto Rico,118.65.126.220,2013-06-02,811876842-2
70,Eric,Murphy,emurphy@voolith.org,Guam,61.204.178.75,2014-01-21,942793522-7
71,Jessica,Fields,jfields@browsezoom.edu,United States of America,209.119.226.121,2013-06-18,087142836-9
72,Louise,Burns,lburns@trudeo.name,Croatia,120.237.78.234,2013-11-21,567098962-7
73,Timothy,Wells,twells@mydo.gov,Hong Kong,114.168.97.201,2013-10-28,730720712-5
74,Brenda,Oliver,boliver@edgeify.mil,Sao Tome and Principe,85.172.173.123,2013-04-29,307446018-0
75,Joseph,Williamson,jwilliamson@centidel.info,New Caledonia,103.211.246.28,2013-11-03,737522725-X
76,Janice,Stanley,jstanley@janyx.mil,Saint Martin,143.112.115.21,2013-02-25,760687094-2
77,William,Bishop,wbishop@camido.edu,Israel,9.207.94.23,2013-09-12,981642962-4
78,Raymond,Sanders,rsanders@oyoyo.biz,Romania,14.23.65.172,2013-10-24,678560037-5
79,Gerald,Ellis,gellis@gigazoom.com,Jordan,7.229.204.172,2013-03-30,400697940-1
80,Jean,Bell,jbell@nlounge.info,French Polynesia,133.203.125.21,2013-07-31,036994112-8
81,Pamela,Lewis,plewis@blogpad.biz,Malaysia,131.144.78.142,2013-08-24,077750278-X
82,Karen,King,kking@yakijo.name,New Caledonia,167.170.85.132,2013-12-14,081115101-8
83,Stephanie,Johnston,sjohnston@katz.info,Bermuda,133.116.75.239,2013-12-29,010221911-7
84,Shawn,Jacobs,sjacobs@zoomzone.gov,Bhutan,171.59.143.115,2013-11-01,479349816-7
85,Ashley,Walker,awalker@quimm.edu,Niger,203.240.122.48,2013-08-26,133415266-7
86,Annie,Henderson,ahenderson@avavee.org,Saint Martin,222.121.65.93,2013-02-14,385710826-6
87,Melissa,Kim,mkim@skippad.biz,Nepal,246.58.173.108,2013-04-20,382410717-1
88,Jonathan,Cox,jcox@gevee.edu,Uruguay,68.245.252.52,2013-09-20,604210327-X
89,Cheryl,Smith,csmith@eamia.org,Gabon,237.122.26.3,2013-10-03,253949390-4
90,Robert,Nichols,rnichols@jayo.net,Norfolk Island,1.63.91.138,2013-12-21,490141423-2
91,Walter,Hunter,whunter@divape.net,Gibraltar,27.113.35.36,2013-06-27,008382412-X
92,Ernest,Gray,egray@tavu.com,Ã…land,179.240.235.23,2013-03-06,209653559-X
93,Irene,Bowman,ibowman@viva.org,Indonesia,42.228.67.11,2013-11-10,274307448-5
94,Denise,Dean,ddean@devcast.net,Gambia,234.17.166.128,2013-06-14,918324928-1
95,Earl,James,ejames@podcat.edu,Serbia,187.235.122.232,2013-09-04,667752076-5
96,Angela,Hansen,ahansen@demivee.com,Heard and McDonald Islands,96.108.249.46,2013-08-02,029936840-8
97,Brandon,Simmons,bsimmons@quinu.net,Namibia,239.101.233.204,2013-06-09,133775501-X
98,Pamela,Johnston,pjohnston@edgeify.info,Sao Tome and Principe,81.169.195.190,2013-03-06,245609692-4
99,Maria,Meyer,mmeyer@browsedrive.name,Indonesia,23.38.206.192,2013-05-22,947365886-9
100,Harry,Davis,hdavis@tekfly.info,Nauru,132.176.225.163,2013-11-27,442034691-4
