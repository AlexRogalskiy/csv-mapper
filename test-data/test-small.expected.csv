constant,rnd,First Lower,first,"Top-Level, Domain",generated
Foo,cccccccccc,t-as,T-AS,567,____5++++++IRAQ0000FFFFFF
Foo,cccccccccc,c-ol,C-OL,345,____3++++++INDO0000FFFFFF
Foo,cccccccccc,j-my,J-MY,234,____2++++++ANTA0000FFFFFF
Foo,cccccccccc,h-en,H-EN,Test,____T++++++NICA0000FFFFFF
Foo,cccccccccc,j-my,J-MY,"[],;""'",____[++++++SOUT0000FFFFFF
Foo,cccccccccc,p-ia,P-IA,123,____1++++++VANU0000FFFFFF
Foo,cccccccccc,j-ne,J-NE,Test,____T++++++PERU0000FFFFFF
Foo,cccccccccc,e-yn,E-YN,234,____2++++++PERU0000FFFFFF
Foo,cccccccccc,s-ah,S-AH,123,____1++++++IREL0000FFFFFF
Foo,cccccccccc,b-by,B-BY,234,____2++++++RWAN0000FFFFFF
Foo,cccccccccc,r-ll,R-LL,567,____5++++++UZBE0000FFFFFF
Foo,cccccccccc,n-le,N-LE,123,____1++++++NEPA0000FFFFFF
Foo,cccccccccc,s-wn,S-WN,789,____7++++++ZAMB0000FFFFFF
Foo,cccccccccc,j-ce,J-CE,T,____T++++++ISLE0000FFFFFF
Foo,cccccccccc,a-my,A-MY,!!!,____!++++++CROA0000FFFFFF
Foo,cccccccccc,v-or,V-OR,234,____2++++++CHRI0000FFFFFF
Foo,cccccccccc,g-ry,G-RY,345,____3++++++VANU0000FFFFFF
Foo,cccccccccc,j-ca,J-CA,234,____2++++++GREE0000FFFFFF
Foo,cccccccccc,s-ve,S-VE,123,____1++++++CAME0000FFFFFF
Foo,cccccccccc,l-is,L-IS,!!!,____!++++++ZAMB0000FFFFFF
Foo,cccccccccc,j-ey,J-EY,345,____3++++++PALE0000FFFFFF
Foo,cccccccccc,t-hy,T-HY,123,____1++++++MARS0000FFFFFF
Foo,cccccccccc,r-th,R-TH,567,____5++++++COOK0000FFFFFF
Foo,cccccccccc,j-ce,J-CE,123,____1++++++BARB0000FFFFFF
Foo,cccccccccc,j-my,J-MY,567,____5++++++SOMA0000FFFFFF
Foo,cccccccccc,j-my,J-MY,234,____2++++++NIGE0000FFFFFF
Foo,cccccccccc,l-an,L-AN,234,____2++++++SLOV0000FFFFFF
Foo,cccccccccc,h-er,H-ER,!!!,____!++++++NIGE0000FFFFFF
Foo,cccccccccc,j-my,J-MY,345,____3++++++KORE0000FFFFFF
Foo,cccccccccc,p-la,P-LA,"[],;""'",____[++++++SWED0000FFFFFF
Foo,cccccccccc,m-le,M-LE,Test,____T++++++CROA0000FFFFFF
Foo,cccccccccc,d-el,D-EL,"[],;""'",____[++++++ANTA0000FFFFFF
Foo,cccccccccc,s-el,S-EL,234,____2+++++++SAN0000FFFFFF
Foo,cccccccccc,l-ra,L-RA,!!!,____!++++++USSR0000FFFFFF
Foo,cccccccccc,v-ia,V-IA,!!!,____!++++++GAMB0000FFFFFF
Foo,cccccccccc,l-is,L-IS,!!!,____!++++++SOLO0000FFFFFF
Foo,cccccccccc,d-hy,D-HY,789,____7++++++UZBE0000FFFFFF
Foo,cccccccccc,f-es,F-ES,Test,____T++++++ANGO0000FFFFFF
Foo,cccccccccc,a-ew,A-EW,!!!,____!++++++NIGE0000FFFFFF
Foo,cccccccccc,r-rd,R-RD,T,____T++++++CAPE0000FFFFFF
Foo,cccccccccc,j-dy,J-DY,234,____2++++++JAMA0000FFFFFF
Foo,cccccccccc,k-ly,K-LY,Test,____T++++++TURK0000FFFFFF
Foo,cccccccccc,d-el,D-EL,123,____1++++++MAUR0000FFFFFF
Foo,cccccccccc,w-er,W-ER,T,____T++++++YUGO0000FFFFFF
Foo,cccccccccc,m-et,M-ET,345,____3++++++KIRI0000FFFFFF
Foo,cccccccccc,k-en,K-EN,345,____3++++++BARB0000FFFFFF
Foo,cccccccccc,a-ce,A-CE,234,____2++++++BRUN0000FFFFFF
Foo,cccccccccc,b-on,B-ON,Test,____T++++++FIJI0000FFFFFF
Foo,cccccccccc,m-sa,M-SA,T,____T++++++SPAI0000FFFFFF
Foo,cccccccccc,p-la,P-LA,T,____T++++++ARUB0000FFFFFF
Foo,cccccccccc,j-se,J-SE,789,____7+++++++SAN0000FFFFFF
Foo,cccccccccc,p-ul,P-UL,567,____5++++++SLOV0000FFFFFF
Foo,cccccccccc,j-my,J-MY,567,____5++++++NORF0000FFFFFF
Foo,cccccccccc,r-th,R-TH,789,____7++++++LESO0000FFFFFF
Foo,cccccccccc,r-an,R-AN,789,____7++++++BENI0000FFFFFF
Foo,cccccccccc,s-an,S-AN,!!!,____!++++++GUER0000FFFFFF
Foo,cccccccccc,p-er,P-ER,123,____1++++++CROA0000FFFFFF
Foo,cccccccccc,k-en,K-EN,123,____1++++++TONG0000FFFFFF
Foo,cccccccccc,b-ly,B-LY,!!!,____!++++++GREE0000FFFFFF
Foo,cccccccccc,t-my,T-MY,789,____7++++++EGYP0000FFFFFF
Foo,cccccccccc,a-io,A-IO,Test,____T++++++TRIN0000FFFFFF
Foo,cccccccccc,s-ey,S-EY,Test,____T++++++PALA0000FFFFFF
Foo,cccccccccc,d-ah,D-AH,"[],;""'",____[++++++EQUA0000FFFFFF
Foo,cccccccccc,w-am,W-AM,345,____3++++++QATA0000FFFFFF
Foo,cccccccccc,r-th,R-TH,345,____3++++++SIER0000FFFFFF
Foo,cccccccccc,v-ia,V-IA,Test,____T++++++ZAMB0000FFFFFF
Foo,cccccccccc,k-th,K-TH,"[],;""'",____[++++++OMAN0000FFFFFF
Foo,cccccccccc,a-da,A-DA,"[],;""'",____[++++++AUST0000FFFFFF
Foo,cccccccccc,r-ph,R-PH,567,____5++++++PUER0000FFFFFF
Foo,cccccccccc,e-ic,E-IC,234,____2++++++GUAM0000FFFFFF
Foo,cccccccccc,j-ca,J-CA,789,____7++++++UNIT0000FFFFFF
Foo,cccccccccc,l-se,L-SE,123,____1++++++CROA0000FFFFFF
Foo,cccccccccc,t-hy,T-HY,Test,____T++++++HONG0000FFFFFF
Foo,cccccccccc,b-da,B-DA,!!!,____!+++++++SAO0000FFFFFF
Foo,cccccccccc,j-ph,J-PH,"[],;""'",____[+++++++NEW0000FFFFFF
Foo,cccccccccc,j-ce,J-CE,!!!,____!++++++SAIN0000FFFFFF
Foo,cccccccccc,w-am,W-AM,789,____7++++++ISRA0000FFFFFF
Foo,cccccccccc,r-nd,R-ND,567,____5++++++ROMA0000FFFFFF
Foo,cccccccccc,g-ld,G-LD,T,____T++++++JORD0000FFFFFF
Foo,cccccccccc,j-an,J-AN,"[],;""'",____[++++++FREN0000FFFFFF
Foo,cccccccccc,p-la,P-LA,567,____5++++++MALA0000FFFFFF
Foo,cccccccccc,k-en,K-EN,123,____1+++++++NEW0000FFFFFF
Foo,cccccccccc,s-ie,S-IE,"[],;""'",____[++++++BERM0000FFFFFF
Foo,cccccccccc,s-wn,S-WN,Test,____T++++++BHUT0000FFFFFF
Foo,cccccccccc,a-ey,A-EY,789,____7++++++NIGE0000FFFFFF
Foo,cccccccccc,a-ie,A-IE,234,____2++++++SAIN0000FFFFFF
Foo,cccccccccc,m-sa,M-SA,567,____5++++++NEPA0000FFFFFF
Foo,cccccccccc,j-an,J-AN,789,____7++++++URUG0000FFFFFF
Foo,cccccccccc,c-yl,C-YL,234,____2++++++GABO0000FFFFFF
Foo,cccccccccc,r-rt,R-RT,345,____3++++++NORF0000FFFFFF
Foo,cccccccccc,w-er,W-ER,345,____3++++++GIBR0000FFFFFF
Foo,cccccccccc,e-st,E-ST,T,____T++++++Ã…LAN0000FFFFFF
Foo,cccccccccc,i-ne,I-NE,234,____2++++++INDO0000FFFFFF
Foo,cccccccccc,d-se,D-SE,345,____3++++++GAMB0000FFFFFF
Foo,cccccccccc,e-rl,E-RL,789,____7++++++SERB0000FFFFFF
Foo,cccccccccc,a-la,A-LA,T,____T++++++HEAR0000FFFFFF
Foo,cccccccccc,b-on,B-ON,345,____3++++++NAMI0000FFFFFF
Foo,cccccccccc,p-la,P-LA,"[],;""'",____[+++++++SAO0000FFFFFF
Foo,cccccccccc,m-ia,M-IA,123,____1++++++INDO0000FFFFFF
Foo,cccccccccc,h-ry,H-RY,"[],;""'",____[++++++NAUR0000FFFFFF